views/registration/registration.hbs

```diff
@@ -37,20 +41,29 @@
 		</div>
 		<p></p>
 		<div id="infotext-u16" class="hidden">
-			<p>{{$t "registration._registration.text.letsTakeThisStepWithYouAndYourChild" (dict "shortTitle" theme.short_title)}}</p>
-			<p>{{$t "registration._registration.text.pleaseRegisterChildThenAgreeToTermsOfUse" (dict "shortTitle" theme.short_title "title" theme.title)}}</p>
+			{{#if cloudConsentNecessary}}
+				<p>{{$t "registration._registration.text.letsTakeThisStepWithYouAndYourChild" (dict "shortTitle" theme.short_title)}}</p>
+				<p>{{$t "registration._registration.text.pleaseRegisterChildThenAgreeToTermsOfUse" (dict "shortTitle" theme.short_title "title" theme.title)}}</p>
+			{{else}}
+				<p>{{$t "registration._registration.text.nextStepRegistration" (dict "title" theme.theme_title)}}</p>
+			{{/if}}
 		</div>
 		<div id="infotext-16" class="hidden">
-			<p>{{$t "registration._registration.text.thankYouForTakingThisStepWithUs" (dict "shortTitle" theme.title)}}</p>
-			<p>{{$t "registration._registration.text.pleaseRegisterFirstAndAgreeToTerms" (dict "themetitle" theme.title)}}</p>
-			<p><strong>{{$t "global.text.attention" }}</strong> {{$t "registration._registration.text.registrationMustBeDoneByYou" }}</p>
+			{{#if cloudConsentNecessary}}
+				<p>{{$t "registration._registration.text.thankYouForTakingThisStepWithUs" (dict "shortTitle" theme.title)}}</p>
+				<p>{{$t "registration._registration.text.pleaseRegisterFirstAndAgreeToTerms" (dict "themetitle" theme.title)}}</p>
```

Comment:
`themetitle` seems to be wrong here, as in the key itself it requires `shortTitle`, did you check if that one works like it is implemented here?

Reply:
was the same before, only some white space is added

Reply:
I know, but if it was wrong before we should fix that as well ;)